src/redux/reducers/reducers.js

```diff
@@ -773,4 +820,8 @@ export default combineReducers({
     curateSearchtext,
     pubmedFetchingMore,
     pubMedCount,
+    AllAdminDepatments,
```

Comment:
reducers file and the list is growing infinitely, let's start using redux slices going forward. For now it's fine to have like this.

Reply:
ok